decoder/vaapidecoder_vp8.cpp

```diff
@@ -406,8 +406,10 @@ VaapiDecoderVP8::VaapiDecoderVP8()
     m_frameHeight = 0;
     m_buffer = 0;
     m_frameSize = 0;
-    memset(&m_frameHdr, 0, sizeof(m_frameHdr));
-    memset(&m_parser, 0, sizeof(Vp8Parser));
+    //memset(&m_frameHdr, 0, sizeof(m_frameHdr));
+    m_frameHdr=Vp8FrameHeader();
```

Comment:
this not needed, since ti's in VaapiDecoderVP8's constructor.  The compile will construct m_frameHdr before call VaapiDecoderVP8()